look-and-feel/css/package.json

```
@@ -16,8 +16,8 @@
     "eslint:fix": "eslint src --ext js,jsx,ts,tsx --fix",
     "stylelint": "stylelint \"src/**/*.{scss,css}\"",
     "stylelint:fix": "stylelint \"src/**/*.{scss,css}\" --fix",
-    "prettier": "prettier \"src/**/*\" --check",
-    "prettier:fix": "prettier \"src/**/*\" --write",
+    "prettier": "prettier \"src/**/*.!(js|jsx|ts|tsx|svg)\" --check",
+    "prettier:fix": "prettier \"src/**/*.!(js|jsx|ts|tsx|svg)\" --write",
```

le (s)css est déjà check par stylelint ce évite les doublon et je ignore-unknown permet de ne pas lever d'érreur pour des fichier que prettier ne prend pas en charge
```suggestion
    "prettier": "prettier \"**/*.!(js|jsx|ts|tsx|scss|css)\" --check --ignore-unknown",
    "prettier:fix": "prettier \"**/*.!(js|jsx|ts|tsx|scss|css)\" --write --ignore-unknown",
```

---

j'ai pas tout capté mais ok je te fais confiance